interfaces/builtin/mount_control.go

```diff
@@ -271,7 +272,15 @@ func enumerateMounts(plug interfaces.Attrer, fn func(mountInfo *MountInfo) error
 	return nil
 }
 
-func validateWhatAttr(what string) error {
+func validateWhatAttr(mountInfo *MountInfo) error {
+	what := mountInfo.what
+
+	// with "functionfs" the "what" can essentially be anything, see
+	// https://www.kernel.org/doc/html/latest/usb/functionfs.html
+	if len(mountInfo.types) == 1 && mountInfo.types[0] == "functionfs" {
+		return apparmor_sandbox.ValidateNoAppArmorRegexp(what)
```

Comment:
this needs to be wrapped in some contextual info I think, see the test, the error looks different from the other